src/DataFixtures/AppFixtures.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\DataFixtures;
+
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+
+class AppFixtures extends Fixture
+{
+    public function load(ObjectManager $manager)
+    {
+        // $product = new Product();
```

Comment:
Wat is dit?

Reply:
@tijsverkoyen is verwijderd in 4d2d3369fb199dc7bd221690898b5d483a09e465 Dit is een voorbeeld die tijdens composer install wordt geplaatst in het project om fixtures te voorzien. Deze zijn nodig als je wilt testen op gitlab